src/mono/wasm/runtime/rollup.config.js

```diff
@@ -40,6 +40,7 @@ const terserConfig = {
         // because of stack walk at src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs
         // and unit test at src\libraries\System.Private.Runtime.InteropServices.JavaScript\tests\timers.js
         keep_fnames: /(mono_wasm_runtime_ready|mono_wasm_fire_debugger_agent_message|mono_wasm_set_timeout_exec)/,
+        keep_classnames: /(ManagedObject|ManagedError|Span|ArraySegment|WasmRootBuffer|SessionOptionsBuilder)/,
```

Comment:
what's the logic we use to decide which ones to keep? is this really the whole set?

Reply:
this are just classes which are public, that is in the `dotnet.d.ts`